compiler/codegen/codegen.cpp

```diff
@@ -1855,6 +1856,16 @@ adjustArgSymbolTypesForIntent(void)
 extern bool printCppLineno;
 debug_data *debug_info=NULL;
 
+
+static const char* getClangBuiltinWrappedName(const char* name)
+{
+  auto it = chplClangBuiltinWrappedFunctions.find(name);
```

Comment:
Since this is not in a HAVE_LLVM ifdef, you can't use C++11 features. Of course, you could put it in such an ifdef...